submanifests/optional.yaml

```diff
@@ -46,10 +48,10 @@ manifest:
       groups:
         - optional
     - name: tflite-micro
-      revision: 1a34dcab41e7e0e667db72d6a40999c1ec9c510c
+      revision: e1eecbee18aa8f7f0ad7e90f8b335564eb52c13f
```

Comment:
same here

Reply:
The branching strategy for `tflite-micro` is the responsibility of @laurenmurphyx64.
Please provide guidance as to whether you would prefer a PR against main or whether you will push as a new branch.

Reply:
I personally have no preference. I haven't been involved in the maintenance of tflite-micro in a long time (probably should remove myself from the collaborator role), and feel it would be better to leave the decision / execution to the folks actively involved in tflite-micro PRs.

Reply:
> same here

Alright, I have made a PR against the `zephyr` branch, someone with permissions on the repo can choose which to use.
https://github.com/zephyrproject-rtos/tflite-micro/pull/5
I wish the reviewers well with the 300k line, 1300 commit PR :)

Reply:
I started reviewing https://github.com/zephyrproject-rtos/tflite-micro/pull/5, I plan to publish that at some time tomorrow (once I finish double cross-checking my notes)

Reply:
I just did an approval there, but I just have a 'shadow'ed' checkmark there...

I noticed that PR has 'merge conflicts' with that branch as the `zephyr` branch had it's own commits stacked on an older commit which was the latest at that time. I dunno if it's possible to merge it through Github....

Last time I updated tflite-micro, I had to ping around and @nashif did the merge of the branch there

Reply:
The whole merge conflicts issue will only get worse with time, hence my preference to go with a clean branch.
Anyway, the original PR was targeting the `zephyr` branch, which was wrong.
See https://github.com/zephyrproject-rtos/tflite-micro/pull/6 for the correct target.

Reply:
I left a change-request review there as there are a few things that are off although I don’t know how much it actually impacts the build.

Reply:
> The whole merge conflicts issue will only get worse with time, hence my preference to go with a clean branch.

Anyone opposed to this approach? I personally prefer starting with clean branches as well.

Now the question is, `zephyr-vZEPHYRVER` vs `vTFLITEMICROVER-zephyr` ...